benchmarks/.ruby-version

```diff
@@ -0,0 +1 @@
+2.7.1
```

Comment:
Ultimately it might be nice to have benchmarks run in multiple versions of Ruby, but for the time being it was easiest for me to just hardcode this to the latest.